### PR TITLE
Tickets/dm 31582

### PR DESCRIPTION
**Pull request by patrickingraham**

### CODE REVIEW

File: python/lsst/ts/externalscripts/auxtel/latiss_cwfs_align.py [hunk 1]
```diff
@@ -588,9 +595,9 @@ def get_schema(cls):
                 type: number
                 default: 0.8
               dataPath:
-                description: Path to the butler data repository.
+                description: Path to the butler data repository on the summit
```
Comment (tribeiro) [inline python/lsst/ts/externalscripts/auxtel/latiss_cwfs_align.py:598]: Maybe mention that this is gen3? Also is it "on the summit" or "at the summit"?

File: python/lsst/ts/externalscripts/auxtel/latiss_cwfs_align.py [hunk 2]
```diff
@@ -373,10 +373,17 @@ async def run_cwfs(self):
             self.log.info(
                 f"Running cwfs in " f"{self.intra_visit_id}/{self.extra_visit_id}."
             )
+            self.log.info(
```
Comment (tribeiro) [inline python/lsst/ts/externalscripts/auxtel/latiss_cwfs_align.py:376]: Consider using debug level for these messages.

File: python/lsst/ts/externalscripts/auxtel/latiss_acquire_and_take_sequence.py [hunk 3]
```diff
@@ -219,7 +219,7 @@ def get_schema(cls):
               dataPath:
                 description: Path to the butler data repository.
```
Comment (tribeiro) [inline python/lsst/ts/externalscripts/auxtel/latiss_acquire_and_take_sequence.py:220]: Maybe mention this is gen3?

File: python/lsst/ts/externalscripts/auxtel/latiss_cwfs_align.py (hunk 1 above)
Comment (patrickingraham) [reply python/lsst/ts/externalscripts/auxtel/latiss_cwfs_align.py:598]: I'd say "on" .... although I think either works...

File: python/lsst/ts/externalscripts/auxtel/latiss_cwfs_align.py (hunk 2 above)
Comment (patrickingraham) [reply python/lsst/ts/externalscripts/auxtel/latiss_cwfs_align.py:376]: I changed a bunch of them throughout and nearby... but actually left this specific instance.

**Review by tribeiro — APPROVED** [review]
Looks good!

File: python/lsst/ts/externalscripts/auxtel/latiss_acquire_and_take_sequence.py (hunk 3 above)
Comment (mfisherlevine) [reply python/lsst/ts/externalscripts/auxtel/latiss_acquire_and_take_sequence.py]: In general, we're trying not to do this anywhere - once Gen3 goes (which will be quite soon now, really) then Gen3 will be the only butler in town. I know this has merged, but just a general not to try not to do this if you can help it.

**Pull request merged**